src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -31,7 +31,7 @@ QString userAgentRawHeaderValue() {
             QStringLiteral("/") +
             VersionStore::version() +
             QStringLiteral(" ( ") +
-            QStringLiteral(MIXXX_WEBSITE_URL) +
+            MIXXX_WEBSITE_URL +
```

Comment:
Can you add a comment why this is required?

Reply:
Force pushed https://github.com/mixxxdj/mixxx/pull/4751/commits/0b7f3527017a62f769a035bfddbd695a64725396 with a comment.